src/static-pages/config.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://github.com/serlo/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import { LanguageCode, ALL_LANGUAGE_CODES } from '../utils'
+
+export const ALL_UNREVISED_TYPES = ['imprint', 'terms'] as const
+export const ALL_REVSIED_TYPES = ['privacy'] as const
+export type UnrevisedType = typeof ALL_UNREVISED_TYPES[number]
+export type RevisedType = typeof ALL_REVSIED_TYPES[number]
+
+const legalRepo = 'https://raw.githubusercontent.com/serlo/serlo.org-legal'
+
+// TODO: i18n
+export const titles: { [K in RevisedType | UnrevisedType]: string } = {
+  imprint: 'Imprint',
+  terms: 'Terms of Use',
+  privacy: 'Privacy'
+}
+
+const config: BaseConfig = {
+  en: {
+    unrevised: {
+      imprint: {
+        url: `${legalRepo}/e391a2407e1191fdd371e255fd33324be933b487/en/imprint.md`
```

Comment:
IMHO: should be `${legalRepo}/master/en/imprint.md` instead. Otherwise, this gets really annoying when we need to change something, especially for the revised stuff (i.e. move away from Markdown or even just change a typo)